### PR TITLE
Feature/bump version to 0.7.4

### PR DESCRIPTION
**Pull request by GavinXu520**
change store into candidate info version is real version

### CODE REVIEW

**Comment by platonci** [comment]
Can one of the admins verify this patch?

**Comment by benbaley** [comment]
ok,to test

**Comment by shinnng** [comment]
ok to test

**Comment by benbaley** [comment]
ok to test

**Comment by shinnng** [comment]
ok to test

**Comment by shinnng** [comment]
add to whitelist

File: x/plugin/staking_plugin.go [hunk 1]
```diff
@@ -2177,10 +2194,10 @@ func (sk *StakingPlugin) ProposalPassedNotify(blockHash common.Hash, blockNumber
 func (sk *StakingPlugin) DeclarePromoteNotify(blockHash common.Hash, blockNumber uint64, nodeId discover.NodeID,
 	programVersion uint32) error {
 
-	version := xutil.CalcVersion(programVersion)
+	//version := xutil.CalcVersion(programVersion)
 
 	log.Debug("Call DeclarePromoteNotify to promote candidate programVersion", "blockNumber", blockNumber,
-		"blockHash", blockHash.Hex(), "real version", programVersion, "calc version", version, "nodeId", nodeId.String())
+		"blockHash", blockHash.Hex(), "real version", programVersion /*, "calc version", version*/, "nodeId", nodeId.String())
```
Comment (cheng762) [inline x/plugin/staking_plugin.go]: suggest commenting a line better than  comment between lines

**Comment by benbaley** [comment]
rerun tests

**Comment by benbaley** [comment]
more discussing needed

**Pull request closed without merging**